ci/resources.yml

```diff
@@ -2,6 +2,17 @@ resources:
   - name: dataworks-analytical-service-infra-pr
     type: pull-request
     source:
-      access_token: ((dataworks-secrets.concourse_github_pat))
-      repository: dwp/dataworks-analytical-service-infra
-    check_every: 1m
+      access_token: ((github_com.token))
+      repo: dwp/dataworks-analytical-service-infra
+      uri: https://github.com/dwp/dataworks-analytical-service-infra.git
+    check_every: 15m
```

Comment:
you won't need the checking, you can use a webhook now

Reply:
Our other repos still use the check - keeping it consistent for now, and will raise a ticket to use webhooks in the future (and retroactively)

Reply:
now that it's in Crown Concourse, you will have to check